### DISCUSSION TITLE
I added my entire root directory to Git and can't figure out how to undo it

### DISCUSSION THREAD

**Discussion by katherinevfry** [nss-evening-cohort-14/evening-client-side#6] [Help Me!] [answered]
**Name:** Katy Fry

**Have you searched the other help tickets?** There aren't any others at this time :)

**Describe your issue**
I somehow managed to commit my ENTIRE Unbutu directory/file to git. I can't figure out how to un-git it. (for lack of a better term)

**List any error messages that appear**
lots of fatal error messages

**What have you googled?** 
https://www.freecodecamp.org/news/how-to-undo-a-git-add/

https://stackoverflow.com/questions/2845731/how-to-uncommit-my-last-commit-in-git

https://scotch.io/tutorials/how-to-use-git-remove-without-deleting-the-files

**What have you tried?** 
`git rm -r --cached <folder>`
`git reset`
`git reset --soft HEAD^`

the `git rm -r` command seems to be what I want but it keeps throwing  error messages like this.
![Screenshot (8)](https://user-images.githubusercontent.com/76188832/104108566-6a29fb00-528b-11eb-9742-aeca2ab7649f.png)

Here's the view from VS code showing that I committed....everything 🆘 
![Screenshot (9)](https://user-images.githubusercontent.com/76188832/104108567-6a29fb00-528b-11eb-97f3-1f609697c0e9.png)

**Comment by TrinityChristiana** [comment]
You're on the right track, but the wrong thing is being Googled. Try googling "How to remove git from a directory".

**Reply by katherinevfry** [reply]
Well that was an easy fix. Thank you so much!

**Reply by drteresavasquez** [reply]
@katherinevfry please provide the fix details below as soon as you can

**Reply by katherinevfry** [reply]
I went to the top of the affected directory and used 

`rm -rf .git`

and it fixed the issue!

**Reply by nashvegasnate** [reply]
Worked for me as well!

**Comment by katherinevfry** [comment, marked as answer]
Solution: I went to the top of the affected directory and used

`rm -rf .git`

and it fixed the issue!

**Reply by GarrettLaw8** [reply]
I did this in class on Saturday and was frantically trying to fix it while we were moving on and couldn't! I finally looked up the correct thing and fixed it but man it freaked me the hell out . I've never felt so helpless!
